nautobot/core/templates/inc/nav_menu.html

```diff
@@ -178,7 +220,23 @@
 
         // Save the scroll position when the navbar is scrolled
         navbar.addEventListener('scroll', debounce(function() {
-            localStorage.setItem('navbarScrollPosition', navbar.scrollTop);
+            sessionStorage.setItem('navbarScrollPosition', navbar.scrollTop);
         }, 250));
+
+         // Add click event listeners to dropdown links
+        addLinkClickListeners();
+
+        // Apply the 'active' class to the previously clicked link
```

Comment:
Can this be updated to only set the clicked link to active if it was previously closed? Currently when you close a dropdown the link remains orange until you click on something else.
![image](https://github.com/nautobot/nautobot/assets/75227981/169ff86f-c0ea-4cab-9edc-0702193eee04)

Reply:
Updated!